int16 number
string time
int16 ground_speed
float64 latitude
float64 longitude
float64 compass
float64 altitude
